src/routes/authentication.js

```diff
@@ -113,11 +115,274 @@ authenticationRouter.post('/signup', validateUserSignup, Agentsignup);
+ *     parameters:
+ *       - name: properties
+ *         description: property object
+ *         in: body
+ *         required: true
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *       - name: Authorization
+ *         in: header
+ *         description: an authentication header
+ *         required: true
+ *         type: string
+ *     responses:
+ *       201:
+ *         description: property uploaded successfully
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
+authenticationRouter.post(
+  '/agent/properties',
+  checkToken,
+  validatePropertyInput,
+  createProperty
+);
+
+/**
+ * @swagger
+ * /properties:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *           type: object
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *         example: {
+ *          "image": game.png,
+ *          "title": "duplex",
+ *          "price": "200000000",
+ *          "address": "abudu abayomi street",
+ *          "city": "lagos",
+ *          "neigbourhood": "ojodu berger",
+ *          "LGA": "ikeja",
+ *          "ZIP": "11011",
+ *          "numberOfBaths": "3",
+ *          "numberofbed": "4",
+ *          "landSize": "4000"
+ *         }
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
 authenticationRouter.get('/properties', getAllProperties);
+
+/**
+ * @swagger
+ * /property/{id}:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: id
+ *         description: id of property
+ *         in: path
+ *         type: integer
+ *         require: true
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *         example: {
+ *          "image": game.png,
+ *          "title": "duplex",
+ *          "price": "200000000",
+ *          "address": "abudu abayomi street",
+ *          "city": "lagos",
+ *          "neigbourhood": "ojodu berger",
+ *          "LGA": "ikeja",
+ *          "ZIP": "11011",
+ *          "numberOfBaths": "3",
+ *          "numberofbed": "4",
+ *          "landSize": "4000"
+ *         }
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
 authenticationRouter.get('/property/:id', getPropertyById);
-authenticationRouter.post('/agent/property', checkToken, validatePropertyInput, createProperty);
+/**
+ * @swagger
+ * /agent/property/{id}:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
```

Comment:
This description is wrong, this endpoint retrieves a single property but you added, retrive all properties from database